### PR TITLE
Add 172 descriptions

### PR DESCRIPTION
**Pull request by rasa**
Almost all of the descriptions come from the websites of the applications.
Replaces #268 which got borked somehow.
Ready for review.
Inspired by https://github.com/lukesampson/scoop-extras/pull/2497

### CODE REVIEW

File: bucket/unar.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://theunarchiver.com/command-line",
+    "description": "The Unarchiver is the only app you need to unrar files.",
```
[the remark this answers is not in the document]
Comment (rasa) [reply bucket/unar.json]: I didn’t write these, I just cut-n-pasted from their websites, or sometimes from Wikipedia, if their site didn’t have anything useful.

File: bucket/sliksvn.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://sliksvn.com/",
+    "description": "A software versioning and revision control system distributed as open source under the Apache License.",
```
Comment (Ash258) [inline bucket/sliksvn.json]:
```suggestion
    "description": "A software versioning and revision control system.",
```

File: bucket/scholdoc.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "http://scholdoc.scholarlymarkdown.com/",
+    "description": "A command-line utility that converts ScholarlyMarkdown documents into the HTML5, LaTeX, and Docx (OOML) formats",
```
Comment (Ash258) [inline bucket/scholdoc.json]:
```suggestion
    "description": "Convert ScholarlyMarkdown documents into the HTML5, LaTeX, and Docx (OOML) formats.",
```

File: bucket/s3deploy.json [hunk 4]
```diff
@@ -1,4 +1,6 @@
 {
+    "homepage": "https://github.com/bep/s3deploy",
+    "description": "A simple tool to deploy static websites to Amazon S3 and CloudFront with Gzip and custom headers support (e.g. \"Cache-Control\")",
```
Comment (Ash258) [inline bucket/s3deploy.json]:
```suggestion
    "description": "Deploy static websites to Amazon S3 and CloudFront with Gzip and custom headers support (e.g. 'Cache-Control')",
```

File: bucket/rktools2k3.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://github.com/kodybrown/rktools2k3",
+    "description": "The binaries from Windows Server 2003 Resource Kit Tools.",
```
Comment (Ash258) [inline bucket/rktools2k3.json]:
```suggestion
    "description": "Streamline manage administrator tasks. (troubleshooting OS, manage AD, deploy applications, ...)",
```

File: bucket/ripgrep.json [hunk 6]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://github.com/BurntSushi/ripgrep",
+    "description": "Recursively searcheS directories for a regex pattern.",
```
Comment (Ash258) [inline bucket/ripgrep.json]:
```suggestion
    "description": "Recursively searches directories for a regex pattern.",
```

File: bucket/restic.json [hunk 7]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://restic.github.io/",
```
Comment (Ash258) [inline bucket/restic.json]: There is static redirect.

```suggestion
    "homepage": "https://restic.net/",
```

File: bucket/restic.json [hunk 8]
```diff
@@ -1,5 +1,6 @@
...
+    "description": "A program that does backups right.",
```
Comment (Ash258) [inline bucket/restic.json]:
```suggestion
    "description": "Fast, secure and simple backup program.",
```

File: bucket/redis.json [hunk 9]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://redis.io",
+    "description": "An open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker.",
```
Comment (Ash258) [inline bucket/redis.json]: No need to mention license in description

```suggestion
    "description": "In-memory data structure store, used as a database, cache and message broker.",
```

File: bucket/pwsh.json [hunk 10]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://github.com/PowerShell/PowerShell",
+    "description": "PowerShell Core is a cross-platform (Windows, Linux, and macOS) automation and configuration tool/framework that works well with your existing tools and is optimized for dealing with structured data (e.g. JSON, CSV, XML, etc.), REST APIs, and object models.",
```
Comment (Ash258) [inline bucket/pwsh.json]:
```suggestion
    "description": "Cross-platform automation and configuration tool/framework, known as Powershell Core, that works well with existing tools and is optimized for dealing with structured data.",
```

File: bucket/portainer.json [hunk 11]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://portainer.io/",
+    "description": "Build and manage your Docker environments with ease today.",
```
Comment (Ash258) [inline bucket/portainer.json]:
```suggestion
    "description": "Simple management solution for Docker.",
```

File: bucket/redis.json (hunk 9 above)
Comment (rasa) [reply bucket/redis.json]: Agreed. I'm just cut-n-pasting, but I don't think the description needs to include:
1. The name of the app (unless it's not the same as the json file).
2. Licensing
3. Open/closed source
4. Command-line or GUI (as we know it's a GUI/CLI by it having a shortcut or not).
5. Marketing fluff ("bestest ever!")

**Comment by rasa** [comment]
@Ash258: Excellent changes. I agree, we don't need description that include:
1. The name of the app (unless it's not the same as the json file).
2. Licensing info (as it's in the license tag)
3. Open/closed source (as it's in the license tag)
4. Command-line or GUI (if it has a shortcut, it's a GUI, otherwise, it's a CLI).
5. Marketing fluff ("simply the best", etc)

File: bucket/graphicsmagick-q16.json [hunk 12]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "http://www.graphicsmagick.org/",
+    "description": "The swiss army knife of image processing. (QuantumDepth of 8 version)",
```
Comment (Ash258) [inline bucket/graphicsmagick-q16.json]:
```suggestion
    "description": "Robust and efficient collection of tools and libraries which support reading, writing, and manipulating an image in over 89 major formats.",
```

File: bucket/graphicsmagick-q8.json [hunk 13]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "http://www.graphicsmagick.org/",
+    "description": "The swiss army knife of image processing. (QuantumDepth of 16 version)",
```
Comment (Ash258) [inline bucket/graphicsmagick-q8.json]:
```suggestion
    "description": "Robust and efficient collection of tools and libraries which support reading, writing, and manipulating an image in over 89 major formats.",
```

File: bucket/hashcat.json [hunk 14]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://hashcat.net/hashcat/",
+    "description": "World's fastest and most advanced password recovery utility.",
```
Comment (Ash258) [inline bucket/hashcat.json]:
```suggestion
    "description": "Advanced password recovery utility.",
```

**Comment by Ash258** [comment]
Please move conversation about description content into lukesampson/scoop#3362. I would like to finish that PR this week and these clarifications are missing in it.

I also want to clarify some of them

File: bucket/hugo-extended.json [hunk 15]
```diff
@@ -1,4 +1,6 @@
 {
+    "homepage": "https://gohugo.io",
+    "description": "The world’s fastest framework for building websites. (extended version)",
```
Comment (Ash258) [inline bucket/hugo-extended.json]:
```suggestion
    "description": "Speed and flexible static site generator with Sass/SCSS support.",
```

File: bucket/hugo.json [hunk 16]
```diff
@@ -1,4 +1,6 @@
 {
+    "homepage": "https://gohugo.io",
+    "description": "The world’s fastest framework for building websites.",
```
Comment (Ash258) [inline bucket/hugo.json]:
```suggestion
    "description": "Speed and flexible static site generator.",
```

**Comment by Ash258** [comment]
Also side note. Please in future, never open such huge PR. keep up with all commits is pain in the ass to do some quality, proper and reliable review.

Chrome is not handling `/files` site with all changes in some good way with rendering all decorators for all files and with refined github, etc, .. :D

**Comment by Ash258** [comment]
As soon as I finish all files it could be merged, but I missed lots of files (some on purpose because it is not 100% clarified how to write proper descriptions), but it could be changed in future.

image

**Comment by Ash258** [comment]
There are also some wrongly resolved conflicts. (two version blocks for example)

image

File: bucket/gnupg.json [hunk 17]
```diff
@@ -10,6 +12,10 @@
         "re": "gnupg-w32-(?<version>[\\d.]+)_(?<date>[\\d]{8}).exe"
     },
     "autoupdate": {
-        "url": "https://www.gnupg.org/ftp/gcrypt/binary/gnupg-w32-$version_$matchDate.exe#/dl.7z"
+        "url": "https://www.gnupg.org/ftp/gcrypt/binary/gnupg-w32-$version_$matchDate.exe#/dl.7z",
+        "hash": {
+            "url": "https://www.gnupg.org/download/integrity_check.html",
+            "regex": "$sha1.*$basename"
```
[the remark this answers is not in the document]
Comment (rasa) [reply bucket/gnupg.json]: Agreed. I just copy-pasted from https://github.com/ScoopInstaller/Main/blob/master/bucket/gpg.json so it and this package were the same.

**Comment by rasa** [comment]
Ok, I will close this and redo, with 25 descriptions at once, OK?

**Pull request closed without merging**